ui/ui-expenses/src/screens/prices/prices.jsx

```diff
@@ -1,21 +1,44 @@
-import React from 'react'
+import React, { Fragment } from 'react'
 import PropTypes from 'prop-types'
 import Price from './price'
+import Who from './who'
 import styles from './prices.styles'
 
-const Prices = ({ prices }) => {
-  const defaultSelected = prices[0]
-  return (
-    <div className={styles.prices}>
+const whos = [
+  {
+    name: 'alakarte',
+    src: '/logo.png',
+  },
+  {
+    name: 'Fabien',
+    src: 'https://avatars1.githubusercontent.com/u/17828231?s=460&v=4',
```

Comment:
is it faster to load a bundled image or a hosted one ?

Reply:
The advantage here is our avatar will be updated in `k-mille` based on github :)

Reply:
yes I understand that 🙂 But I keep asking my question (for personal culture 😅)

Reply:
I think this is even if we don't use HTTP2 push technology.

If we use HTTP2 push, then this is faster to have images hosted with static files (JS/CSS) since the server can "push" in the same connexion.